### PR TITLE
improvement(custom_d1): use greatly improved latte+rune scripts

### PR DESCRIPTION
**Pull request by vponomaryov**
### Testing

- 3h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#8](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=cb36d00e-6b3d-4342-a709-eb93e9a9c9b8) (PASSED)
- 12h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#10](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f92f22e5-c0b4-4d80-9ea0-7e2785d749e7) (PASSED)

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevent to this change (if needed)

### CODE REVIEW

**Comment by vponomaryov** [comment]
@fruch it is workable, just intermediate state. Will update it according to our discussion.

File: test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml [hunk 1]
```diff
@@ -3,24 +3,84 @@ n_db_nodes: 6
...
+    --function custom -P row_count=164572800
+    -P print_applied_func_names=2 -P codes="\"T4F2\""
+    scylla-qa-internal/custom_d1/workload1/latte/custom_d1_workload1.rn
 round_robin: true
 
-nemesis_during_prepare: false # latte load phase is not very good with retries yet
+# TODO: enable 'nemesis_during_prepare' back when debug of stress commands get finished
+nemesis_during_prepare: false
 
 gce_instance_type_db: 'n2-highmem-8'
```
Comment (fruch) [inline test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml:79]: we should increase it to n2-highmem-32

**Review by fruch — COMMENTED** [review]
Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

**Comment by vponomaryov** [comment]
> Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

[scylla-staging/valerii/vp-longevity-d1-hybrid-raid-n2-highmem-32-2022.1#15](https://argus.scylladb.com/test/db271c9f-b07d-4644-8df5-ad368d9f3a5c/runs?additionalRuns[]=f0793996-2a3e-453d-ae84-bd6e83534cd5)

File: test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml (hunk 1 above)
Comment (vponomaryov) [reply test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml:79]: I am working on it.

Comment (vponomaryov) [reply test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml]: Done

**Comment by vponomaryov** [comment]
> Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

- 3h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#8](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=cb36d00e-6b3d-4342-a709-eb93e9a9c9b8) (PASSED)
- 12h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#10](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f92f22e5-c0b4-4d80-9ea0-7e2785d749e7) (PASSED)

**Comment by vponomaryov** [comment]
It is being tested here: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#15](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f01a2dd0-fe64-4eaf-8194-312c143bb057)

So, this is really working solution.
Open item is to pick up some nemesis_seed.
From my point of view, it is ready to be merged.

**Comment by vponomaryov** [comment]
@roydahan , @fruch This PR is waiting for you

**Comment by roydahan** [comment]
I won't be able to reach it before the week after the summit.

**Comment by vponomaryov** [comment]
> @roydahan , @fruch This PR is waiting for you

Still true

**Review by roydahan — APPROVED** [review]
I'm not into the details of the customer profile in order to really review the latte command line parameters but judging from the test results and reviewing everything else, it looks good to me.

**Comment by vponomaryov** [comment]
Changes:
- Updated the docker images to contain fix for memory leaks
- Switched the instance type for loaders to cheaper ones - `e2-highcpu-32` instead of the `e2-standard-32`.

**Comment by fruch** [comment]
@vponomaryov need to fix those

**Comment by vponomaryov** [comment]
> @vponomaryov need to fix those

Done

**Pull request merged**